crates/wast/src/ast/mod.rs

```diff
@@ -62,7 +62,7 @@ macro_rules! custom_keyword {
         }
 
         impl $crate::parser::Peek for $name {
-            fn peek(cursor: $crate::parser::Cursor<'_>) -> bool {
+            fn peek(cursor: $crate::parser::Cursor<'_>) -> std::primitive::bool {
```

Comment:
FWIW I think it's fine below to do something like `bool_ = "bool"` to avoid the need for this. I don't think the usage of raw identifiers to avoid clashes with rust identifiers has really worked out all that well and at some point I should probably move everything that's a rust identifier to an underscore after it as well. Anyway minor point, could go either way here.